### DISCUSSION TITLE
Obsolete techniques - 6rd, site-local

### DISCUSSION THREAD

**Discussion by EduardVasilenko** [becarpenter/book6#41] [General]
Hi Brian,
I propose to mention 6rd (RFC 5569) just after "6to4" because like the latter it is not the path for the transition to the future.

It is possible to mention "site-local" deprecated by RFC 3879.
Ed/

**Comment by becarpenter** [comment]
Agree for 6rd, will fix.

I will note "site-local" in 2. Addresses in a new sub-section on obsolete formats.

**Comment by becarpenter** [comment]
Closed as completed months ago
